feature_categories/src/main/java/cm/aptoide/pt/feature_categories/domain/CategoriesUseCase.kt

```diff
@@ -1,15 +1,17 @@
 package cm.aptoide.pt.feature_categories.domain
 
+import cm.aptoide.pt.aptoide_network.di.StoreName
 import cm.aptoide.pt.aptoide_network.domain.UrlsCache
 import cm.aptoide.pt.feature_categories.data.CategoriesRepository
 import javax.inject.Inject
 
 class CategoriesUseCase @Inject constructor(
   private val urlsCache: UrlsCache,
   private val categoriesRepository: CategoriesRepository,
+  @StoreName private val storeName: String,
 ) {
-  suspend fun getCategories(tag: String): List<Category> =
-    urlsCache.get(id = tag)
-      ?.let { categoriesRepository.getCategoriesList(it) }
-      ?: throw IllegalStateException("No url found")
+  suspend fun getCategories(tag: String): List<Category> {
+    val url = urlsCache.get(id = tag) ?: "store/groups/get/store_name=$storeName/groups_depth=1/group_name=games/hide_empty=1/hide_not_foreign=1/limit=9/sort=score"
```

Comment:
This here limits the categories listing to 9 categories, we don't want this as some categories won't appear this way. (For now it seems fine because we only have 9 non-empty categories, but it won't be like this forever)